# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/muffin/no-logind.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- src/backends/native/meta-launcher.c.vanilla	2024-09-06 23:59:06.959518379 +0200
+++ src/backends/native/meta-launcher.c	2024-09-06 23:59:11.898559973 +0200
@@ -31,7 +31,6 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
-#include <systemd/sd-login.h>
 #include <gudev/gudev.h>
 
 #include "backends/meta-backend-private.h"
--- src/core/main.c.vanilla	2024-09-06 23:57:20.186617375 +0200
+++ src/core/main.c	2024-09-06 23:58:45.779339932 +0200
@@ -67,7 +67,7 @@
 #endif
 
 #if defined(HAVE_NATIVE_BACKEND) && defined(HAVE_WAYLAND)
-#include <systemd/sd-login.h>
+//#include <systemd/sd-login.h>
 #endif /* HAVE_WAYLAND && HAVE_NATIVE_BACKEND */
 
 #ifdef HAVE_SYS_PRCTL
--- src/meson.build.vanilla	2024-09-06 23:54:29.741171376 +0200
+++ src/meson.build	2024-09-06 23:54:40.073259310 +0200
@@ -123,7 +123,6 @@
     libinput_dep,
     gudev_dep,
     libgbm_dep,
-    logind_provider_dep,
     libudev_dep,
     xkbcommon_dep,
   ]
--- meson.build.vanilla	2024-09-06 23:49:43.635780471 +0200
+++ meson.build	2024-09-06 23:54:06.869976590 +0200
@@ -1,4 +1,4 @@
-project('muffin', 'c', version : '6.2.0', meson_version : '>=0.56.0', license : 'GPLv2+')
+project('muffin', 'c', version : '7.2.0', meson_version : '>=0.56.0', license : 'GPLv2+')
 
 mutter_plugin_api_version = '3'
 
@@ -184,13 +184,6 @@
   libgbm_dep = dependency('gbm', version: gbm_req)
   libinput_dep = dependency('libinput', version: libinput_req)
 
-  libsystemd_dep = dependency('libsystemd', required: false)
-  if libsystemd_dep.found()
-    logind_provider_dep = libsystemd_dep
-  else
-    logind_provider_dep = dependency('libelogind')
-  endif
-
   if not have_egl
     error('The native backend requires EGL to be enabled')
   endif
